### PR TITLE
Add blog post about the meet up.

### PR DESCRIPTION
**Pull request by parkr**
@gjtorikian rocks.

/cc @benbalter @jglovier @jekyll/core 

Let me know if this looks good. :heart:

### CODE REVIEW

File: site/_posts/2015-01-20-jekyll-meet-and-greet.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+layout: news_item
+title: "Jekyll Meet & Greet at GitHub HQ"
+date: "2015-01-20 19:23:12 -0800"
+author: parkr
+categories: [meetup]
+---
+
+Hey! Our friends at GitHub have agreed to host a Jekyll meet & greet on
+**February 5, 2015 at 7pm**. The event will be hosted at [GitHub's
+Headquarters](https://www.google.com/maps/place/GitHub/@37.782084,-122.391403)
```
Comment (gjtorikian) [inline site/_posts/2015-01-20-jekyll-meet-and-greet.markdown]: We should switch this to point to 85 Colin P. Kelley, which is the visitor entrance (people will need to come through here anyway): http://goo.gl/qMeBi5

Comment (parkr) [inline site/_posts/2015-01-20-jekyll-meet-and-greet.markdown]: Ooooh yes good point. :smile_cat:

**Comment by alfredxing** [comment]
:+1: Wish I could be there though :cry:

File: site/_posts/2015-01-20-jekyll-meet-and-greet.markdown (hunk 1 above)
Comment (alfredxing) [inline site/_posts/2015-01-20-jekyll-meet-and-greet.markdown]: Correct me if I'm wrong, but according to [various GitHub meetup posts](https://github.com/blog/search?q=Colin+P+Kelly) and [a Google Street View image featuring your awesome Octocat sign](https://www.google.com/maps/@37.78235,-122.391131,3a,50.7y,225.92h,86.66t/data=!3m4!1e1!3m2!1sLHQUzmqQ3GAE2u1JzPBWmg!2e0), is GitHub's address 88 Colin P. Kelly (not 85)?

Comment (envygeeks) [inline site/_posts/2015-01-20-jekyll-meet-and-greet.markdown]: I would go but there are some logistical problems:

1. There is no spot for me to park to display my my George Bush sticker.
2. I don't think I could get my Hummer H1 around the corner... corner parkers.

Comment (envygeeks) [inline site/_posts/2015-01-20-jekyll-meet-and-greet.markdown]: Oh, and there is on last problem, I don't think I could book the flight, the Hummer and find the sticker I need to do my obligatory trolling when visiting San Francisco in time, but I do wish you guys the best with your meetup!

**Comment by jglovier** [comment]
:metal:

**Comment by alfredxing** [comment]
@gjtorikian @jglover Correct me if I'm wrong, but according to [various GitHub meetup posts](https://github.com/blog/search?q=Colin+P+Kelly) and [a Google Street View image featuring your awesome Octocat sign](https://www.google.com/maps/@37.78235,-122.391131,3a,50.7y,225.92h,86.66t/data=!3m4!1e1!3m2!1sLHQUzmqQ3GAE2u1JzPBWmg!2e0), is GitHub's address 88 Colin P. Kelly (not 85)?

Link to Google Map: https://goo.gl/maps/nIi9D

**Comment by jglovier** [comment]
@alfredxing yeah, good catch. The Google maps link in the post is incorrect. I'll open a pull to fix.

**Pull request merged**